homework/kostyantyn-gamanyuk/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1></h1>
+    <ul>
+        <li><a href="/lesson-07/build/index.html">07 Lesson</a></li>
+        <li><a href=""></a></li>
```

Comment:
if there is no url in the link yet, add **#** instead of it
like this:
`<li><a href="#"></a></li>`